src/components/Card.tsx

```diff
@@ -3,11 +3,11 @@ import { FC } from 'react';
 import classes from './Card.module.css';
 import { CardProps } from './Card.types';
 
-const Card: FC<CardProps> = ({ id, title, image, text, tags, isDragging }): React.ReactElement => {
+const Card: FC<CardProps> = ({ id, title, image, text, tags, isDragging, onClick }): React.ReactElement => {
```

Comment:
I'd advice to use another name for click handler, to improve readability. clickHandler or sometimes we can even go deeper into domain specifics. In this case `ckickHandler` or `handleClick` is good enough.